.env.example

```diff
@@ -9,3 +9,20 @@ SPREADSHEET_ID="1jlILhCXWB9wS2A3DMYZ2ZARGWlUBss4-u1JneTM0Owg"
 # for example if it was inside **utils** folder you should add it like this:
 # "utils/google_service_account-steady-scope-436117-p2-f407b01106c9.json"
 GOOGLE_SERVICE_ACCOUNT_JSON_FILE="google_service_account-arctic-diode-438812-b5-7338724a2dea.json"
+
+
+# USE THIS ONLY IF DEPLOYING TO Vercel, Heroku app, etc
+# Add into this variable the whole object inside Service Account's JSON file.
+# Should look something like this:
+GOOGLE_SERVICE_ACCOUNT_CREDENTIALS=`{
+  "type": "service_account",
+  "project_id": "your-project-id",
+  "private_key_id": "your-private-key-id",
+  "private_key": "your-private-key",
+  "client_email": "your-client-email",
+  "client_id": "your-client-id",
+  "auth_uri": "https://accounts.google.com/o/oauth2/auth",
+  "token_uri": "https://accounts.google.com/o/oauth2/token",
+  "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+  "client_x509_cert_url": "your-cert-url"
+}`
```

Comment:
Hyvä idea oli tuo objektin lisääminen `.env`-tiedostoon

Olisiko hyvä siirtyä kaiken kohdalla tähän tapaan vai pidetäänkö tämä vain, jos lisää ohjelman Verceliin, Herokuun tmv?

`# USE THIS ONLY IF DEPLOYING TO Vercel, Heroku app, etc` tämä rivi ei sinänsä pidä paikkaansa, koska tämä toimii ihan hyvin omallakin koneella :smile: 

Jos haluaa lukasta enemmän, niin tuolta löytyy: https://cloud.google.com/nodejs/docs/reference/google-auth-library/latest#loading-credentials-from-environment-variables

Reply:
Jättäisin molemmat tavat enkä rajoittaisi toista Verceliin. Tärkeää on mainita kumpaa priorisoidaan jos molemmat annettu.

Reply:
Varsinaista nopeuseroa ei noissa kahdessa tavassa ollut

Voisin laittaa näin: jos `GOOGLE_SERVICE_ACCOUNT_CREDENTIALS` on määritetty, käytetään sitä, muuten JSON-tiedostoa :+1: